owasp-dependency-check-suppressions.xml

```diff
@@ -275,18 +275,44 @@
   </suppress>
 
   <suppress>
-    <!-- We need to wait for 17.0.0 of https://github.com/kubernetes-client/java/releases -->
-    <!-- We need to update several other components to move to Snakeyaml 2.0 to address CVE-2022-1471 -->
-    <!-- Snakeyaml 1.33 added to dependencyManagement in main pom file -->
+    <!-- The main use of snakeyaml in Druid is coming in test scope from:
+     com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:jar:2.12.7
+     (version 1.27)
+     The contrib extension: druid-cassandra-storage uses version 1.6 in compile
+     scope
+     The integration tests use version 1.27 in compile scope.
```

Comment:
do we know where 1.27 comes from in integration tests? It would be nice if we could upgrade integration tests to 2.x as well.

Reply:
it's transitive dependency of: 
com.fasterxml.jackson.dataformat:jackson-dataformat-yaml

Reply:
let's update the version since we pin integration tests to 1.33 and not 1.27

Reply:
if this is more hairy, we can do the jackson upgrade as a follow-up